assignment_02/resources/views/student/edit.blade.php

```diff
@@ -0,0 +1,76 @@
+
+@section('content')
+<div class="container">
+    <div class=" mt-3">
+        @if (session('success'))
+        <div class="alert alert-success alert-dismissible fade show" role="alert">
+            <span class="text-center">{{session('success')}}</span>
+            <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
+        </div>
+        @endif
+    </div>
+    <div class="card mt-5">
+        <h4 class="card-header font-weight-bold">
+            Student Edit
+        </h4>
+        <div class="card-body">
+            <form action="{{route('student#update',$student->id)}}" method="POST">
+                @csrf
+                <div class="form-group">
+                    <label for="">Name</label>
+                    <input type="text" class="form-control @error('name') is-invalid @enderror" id="" placeholder="name" name="name" value="{{$student->name}}">
+                    @error('name')
+                    <div class="invalid-feedback">{{$message}}</div>
+                    @enderror
+                </div>
+
+                <div class="form-group">
+                    <label for="">Major</label>
+                    <select name="major_id" required class="form-control @error('major_id') is-invalid @enderror">
+                        @error('major_id')
+                        <div class="invalid-feedback">{{$message}}</div>
+                        @enderror
+                        @foreach($majors as $major)
+                        <option value="{{ $major->id }}" @if ($student->major_id==$major->id) selected @endif>{{ $major->name }}</option>
+                        @endforeach
+                    </select>
+                </div>
+
+                <div class="form-group">
+                    <label for="">Phone</label>
+                    <input type="number" class="form-control @error('phone') is-invalid @enderror" id="" placeholder="09***********" name="phone" value="{{$student->phone}}">
+                    @error('phone')
+                    <div class="invalid-feedback">{{$message}}</div>
+                    @enderror
+                </div>
+
+                <div class="form-group">
+                    <label for="">Email</label>
+                    <input type="text" class="form-control @error('email') is-invalid @enderror" id="" placeholder="name@example.com" name="email" value="{{$student->email}}">
+                    @error('email')
+                    <div class="invalid-feedback">{{$message}}</div>
+                    @enderror
+                </div>
+
+                <div class="form-group">
+                    <label for="">Address</label>
+                    <textarea class="form-control @error('address') is-invalid @enderror" id="" name="address">{{$student->address}}</textarea>
+                    @error('address')
+                    <div class="invalid-feedback">{{$message}}</div>
+                    @enderror
+                </div>
```

Comment:
These codes seem to be the same as in student/create.blade.
Try to use sub view. 
https://laravel.com/docs/10.x/blade#including-subviews
- Create sub view including common input elements and share that view file from create and eidt blades.